### ISSUE TITLE
chaining schedulers

### ISSUE THREAD

**Issue by ruotianluo** [PyTorchLightning/pytorch-lightning#7662]
## 🚀 Feature
Currently (to the best of my understanding), each optimizer can only paired with one lr scheduler. As shown in the official pytorch doc(https://pytorch.org/docs/master/optim.html), one can actually chain two schedulers with one optimizer.

### Motivation

I want to have two lr_scheduler, that is a linear warmup(with iteration) followed by a epoch-wise decay. So I need two lr_scheduler where one scheduler has interval to be 'step' and another to be 'epoch'.

### Pitch

So, basically I am expecting to have a configure_optimizer to return:

```
return {
        'optimizer': optimizer,
        'lr_scheduler': [{
            'scheduler': LambdaLR(optimizer, ...),
           'interval': 'step'
        },
        {
            'scheduler': LambdaLR(optimizer, ...),
            'interval', 'epoch'
        }
        ]
    }
```

**Comment by Lucklyric** [comment]
I think this is a bit relevant to this https://github.com/PyTorchLightning/pytorch-lightning/issues/7553#issue-892168319 . I think a simplest way to do that is to have a lr_scheduler wrapper. However, it might need some investigation on how to handle different behaviours between different lr_schedulers. E.g. some sub-schedulers might need to monitor a metric and others might not.
